internal/schema/set_map.go

```diff
@@ -0,0 +1,36 @@
+package utils
```

Comment:
nit: we can correct this later as minor cleanup. 
the package name should match -- schema

underscores in filenames can be interpreted as compiler hints, as in `_arm64.go`. I try to avoid them. 
In this case, maps.go could be an alternative.